### PR TITLE
updates gem dependencies

### PR DESCRIPTION
**Pull request by LeFnord**

### CODE REVIEW

**Comment by LeFnord** [comment]
@dm1try, @namusyaka or @dblock … can anybody have an eye on, and maybe mörge?

**Comment by namusyaka** [comment]
Good catch!  I'm going to review your changes today

**Comment by LeFnord** [comment]
@namusyaka … fine, so I can push the next one 😉

File: lib/grape/router/attribute_translator.rb [hunk 1]
```diff
@@ -13,8 +13,10 @@ def to_h
       def method_missing(m, *args)
         if m[-1] == '='
           @attributes[m[0..-1]] = *args
-        else
+        elsif m[-1] != '='
           @attributes[m]
+        else
+          super
```
Comment (namusyaka) [inline lib/grape/router/attribute_translator.rb]: Is this on purpose? Is it possible to reach this `super`?

File: lib/grape/dsl/helpers.rb [hunk 2]
```diff
@@ -31,9 +31,11 @@ def helpers(new_mod = nil, &block)
...
             define_boolean_in_mod(mod)
             inject_api_helpers_to_mod(mod) if new_mod
-            inject_api_helpers_to_mod(mod) do
-              mod.class_eval(&block)
-            end if block_given?
+            if block_given?
+              inject_api_helpers_to_mod(mod) do
+                mod.class_eval(&block)
+              end
+            end
```
Comment (namusyaka) [inline lib/grape/dsl/helpers.rb]: Why do you hate modifier `if` with block or multiple lines ?

File: gemfiles/rails_edge.gemfile [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# This file was generated by Appraisal
+
+source 'https://rubygems.org'
+
+gem 'rails', '~> 5.0.0'
```
Comment (namusyaka) [inline gemfiles/rails_edge.gemfile]: Should we add this `appraise` into `Appraisal`? And I guess we should use edge of github.
it should be something like `gem 'rails', github: 'rails/rails'`

**Comment by namusyaka** [comment]
@LeFnord I pointed out a couple of points. Thanks for your improvement!

File: lib/grape/dsl/helpers.rb (hunk 2 above)
Comment (LeFnord) [inline lib/grape/dsl/helpers.rb]: I don't do it, it seems rubocop hate it 😬 … wil add a cop which allows it

File: gemfiles/rails_edge.gemfile (hunk 3 above)
Comment (LeFnord) [inline gemfiles/rails_edge.gemfile]: good point

File: lib/grape/router/attribute_translator.rb (hunk 1 above)
Comment (LeFnord) [inline lib/grape/router/attribute_translator.rb]: ti should not be reachable, but again rubocop understand it -> will add the cop

File: gemfiles/rails_edge.gemfile (hunk 3 above)
Comment (LeFnord) [inline gemfiles/rails_edge.gemfile]: have it, but didn't used it before, if you have expierience with it …

**Comment by LeFnord** [comment]
@namusyaka … comments addressed

**Comment by namusyaka** [comment]
Looks good to me

**Comment by LeFnord** [comment]
@namusyaka … feel free and trust yourself to mörge it

**Comment by dblock** [comment]
Merged. Looks good.

**Pull request merged**